typings/globals.d.ts

```diff
@@ -1,7 +1,12 @@
 declare var BlobBuilder: any;
+declare var errorWrapper: any;
```

Comment:
I'm not sure some of these should be here. I can understand its use for external libs (BlobBuilder, MIDI, Guppy, Sk) but the rest are Oppia-specific right? We should be dealing with the Oppia-specific variables internally.

Reply:
`errorWrapper` is removed now. For other variables which are required when files are compiled as standalone, I have moved them to a separate declaration file.